modules/mobile/spec/request/decision_letters_request_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+require_relative '../support/matchers/json_schema_matcher'
+require Rails.root.join('modules', 'claims_api', 'spec', 'support', 'fake_vbms.rb')
+
+RSpec.describe 'decision letters', type: :request do
+  include JsonSchemaMatchers
+
+  before do
+    allow(VBMS::Client).to receive(:from_env_vars).and_return(FakeVBMS.new)
+    allow_any_instance_of(IAMUser).to receive(:icn).and_return('24811694708759028')
+    iam_sign_in(build(:iam_user))
+  end
+
+  # This endpoint's upstream service mocks it's own data for test env. HTTP client is not exposed by the
+  # connect_vbms gem so it cannot intercept the actual HTTP request, making the use of VCRs not possible.
+  # This means we cannot test error states for the index endpoint within specs
+  describe 'GET /mobile/v0/decision_letters' do
```

Comment:
Is it worth adding tests for invalid responses?

Reply:
unfortunately the upstream service doesn't have the ability to be used with VCRs so it just directly mocks the data. so the only response we can get out of it is valid data in test env. I played with manually created a faraday bad response but wasn't getting anywhere with it.